modules/mgmt-power-remediation-baremetal-about.adoc

```diff
@@ -0,0 +1,114 @@
+// Module included in the following assemblies:
+
+// * machine_management/mgmt-power-remediation-baremetal
+
+[id="mgmt-power-remediation-baremetal-about_{context}"]
+= About power-based remediation of bare metal
+In a bare metal cluster, remediation of nodes is critical to ensure the overall health of the cluster. Physically remediating a cluster can be challenging and any delay in putting the machine to a safe or an operational state increases the time the cluster remains in a degraded state, and the risk that subsequent failures might bring the cluster offline. Power-based remediation helps counter such challenges.
+
+Instead of reprovisioning the nodes, power-based remediation uses a power controller to power off an inoperable node. This type of remediation is also called power fencing.
+
+The {product-title} uses the `MachineHealthCheck` controller to detect faulty bare metal nodes by initiating power-cycling. Power-based remediation is fast and reboots unhealthy hosts instead of removing them from the cluster.
+
+Power-based remediation meets the following objectives:
+
+* Allows the recovery of control plane nodes
+* Prevents data loss in hyperconverged environments.
```

Comment:
If the node was left running, then in theory it could be generating or processing data that isn't being synchronised to the rest of the cluster.  Which would count as data loss.

Likewise if we recover a master, the cluster can survive the subsequent loss of another one - allowing incoming data to be process and not lost.

So technically true, but possibly a little misleading.

```suggestion
* Reduces the risk data loss in hyperconverged environments.
* Reduces the downtime associated with recovering physical machines
```